controllers/admin/AdminThemesController.php

```diff
@@ -785,6 +785,7 @@ public function processExportTheme()
                         ->from('hook', 'h')
                         ->leftJoin('hook_module', 'hm', 'hm.`id_hook` = h.`id_hook`')
                         ->leftJoin('module', 'm', 'hm.`id_module` = m.`id_module`')
+			->leftOuterJoin('hook_module_exceptions', 'hme', 'hme.`id_module` = hm.`id_module` AND hme.`id_hook` = h.`id_hook`')
```

Comment:
Can you fix this indentation to spaces please?

Reply:
I should be able to